src/compiler/diagnosticMessages.json

```
@@ -3641,6 +3641,26 @@
         "category": "Error",
         "code": 4111
     },
+    "This member cannot have an 'override' modifier because its containing class '{0}' does not extend another class.": {
+        "category": "Error",
+        "code": 4112
+    },
+    "This member cannot have an 'override' modifier because it is not declared in the base class '{0}'.": {
+        "category": "Error",
+        "code": 4113
+    },
+    "This member must have an 'override' modifier because it overrides a member in the base class '{0}'.": {
+        "category": "Error",
+        "code": 4114
+    },
+    "This parameter property must be rewritten as a property declaration with an 'override' modifier because it overrides a member in base class '{0}'.": {
+        "category": "Error",
+        "code": 4115
+    },
+    "This member must have an 'override' modifier because it is override an abstract method that is declared in the base class '{0}'.": {

```

```suggestion
    "This member must have an 'override' modifier because it is overrides an abstract method that is declared in the base class '{0}'.": {

```

---

Uh, let me take a stab

```suggestion
    "This member must have an 'override' modifier because it overrides an abstract method that is declared in the base class '{0}'.": {
```

---

Oops, even my correction had a typo. Thanks!